src/addfield.F90

```diff
@@ -269,12 +281,12 @@ subroutine addfield(cgrib, lcgrib, ipdsnum, ipdstmpl, ipdstmplen, &
   if (ibmap .eq. 0 .or. ibmap .eq. 254) then
      allocate(pfld(max(2, ngrdpts)))
      ndpts = 0;
-     do jj = 1, ngrdpts
```

Comment:
why this change?

Reply:
'jj' was being used as that iterator and was being changed by the outputs from the call here:
https://github.com/NOAA-EMC/NCEPLIBS-g2/blob/da8a6c538afb6b4a38e778b8513220d2f11f7010/src/addfield.F90#L313-L315

I didn't want to risk any problems by having two of the same variable name used two different ways.

Reply:
Are we sure this does not change code behavior? (Even if that behavior is a bug?)

Reply:
the only use of `jj` is in the three lines i showed previous. The new iterator `ii` is not used elsewhere in the code. In this case the code was fine, there was no cause for concern about reusing `jj` like it was. It's just not smart to have a variable used in two different ways like that.